src/components/QueryOutput/QueryOutput.js

```diff
@@ -851,7 +877,87 @@ export class QueryOutput extends React.Component {
 
     return undefined
   }
-
+  getTrainingData = () => {
+    const isYScaleStringColumn = !this.chartRef?.innerChartRef?.yScale.fields[0].hasOwnProperty('aggType')
+    const isXScaleStringColumn = !this.chartRef?.innerChartRef?.xScale.fields[0].hasOwnProperty('aggType')
+    const isTableFilterInXScale = this.queryResponse?.data?.data?.fe_req.filters.some(
+      (obj) => obj.name === this.chartRef?.innerChartRef?.xScale.column.name,
+    )
+    const isTableFilterInYScale = this.queryResponse?.data?.data?.fe_req.filters.some(
+      (obj) => obj.name === this.chartRef?.innerChartRef?.yScale.column.name,
+    )
```

Comment:
What if there is a second y axis? ie. a column_line combo chart. Or a pie chart with no axes?